### ISSUE TITLE
Collection::unsearchable() does not delete objects from my index.

### ISSUE THREAD

**Issue by mozammil** [algolia/scout-extended#142]
Hi @nunomaduro 

The `Algolia\ScoutExtended\Jobs\DeleteJob` uses `deleteBy` internally to make a Collection unsearchable. 

```
$result = $index->deleteBy([
    'tagFilters' => $this->searchables->map(function ($searchable) {
        return ObjectIdEncrypter::encrypt($searchable);
    })->toArray(),
]);
```
However, based on the [documentation](https://www.algolia.com/doc/api-reference/api-parameters/tagFilters/), this will use the conjunction **AND** to find the items, whereas we are probably looking for **OR**. 

I just gave this a try by sending a batch delete job and it couldn't delete anything because it can't find records that have these filters: `App\Movie:1, App::Movie:2...` which makes sense. 

This is what I tried: 

```
$movies = \App\Movie::get();
$movies->unsearchable();
```

**Edit**: I was able to fix the issue by using a nested array for Tag Filters. See [this](https://github.com/mozammil/scout-extended/blob/fix-bug-to-allow-for-batch-deletes/src/Jobs/DeleteJob.php#L56). I am not entirely sure what are the implications for doing this, to be honest. If you feel comfortable with this, I'd happily submit a PR for it.

**Comment by nunomaduro** [comment]
Yes. You are right. Well spoted!

Can you perform a Pull Request, and update tests please?

**Comment by nunomaduro** [comment]
This issue was being fixed on the version `v1.3.1`. Thank you!
